simplejira/wrapper.py

```diff
@@ -315,11 +315,14 @@ def search_issues(self, assignee=None, sprint=None, status=None, text=None):
         Returns:
             List of JIRA.Issue resources
         """
-        if not sprint:
-            sprint = self.current_sprint_id
+        if sprint == "backlog":
+            search_query = 'project = RHCFQE AND sprint is EMPTY'
```

Comment:
We don't want to hard code project. You can use 'self.project_id' in the wrapper to get current project ID. I think the search will work with the id number as well?

Reply:
(project_id is a property ... look for 'def project_id')

Reply:
Updated. Please review.